### PR TITLE
manually create the hstore extension for pulpcore externaldb

### PR DESCRIPTION
**Pull request by evgeni**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/proc_installing-postgresql.adoc [hunk 1]
```diff
@@ -96,6 +96,16 @@ CREATE DATABASE candlepin OWNER candlepin;
...
+. Connect to the Pulp database and create the `hstore` extension:
++
+[options="nowrap" subs="verbatim,quotes"]
+----
+postgres=# \c pulpcore
+You are now connected to database "pulpcore" as user "postgres".
+pulpcore=# CREATE EXTENSION IF NOT EXISTS "hstore";
+CREATE EXTENSION
+----
++
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-postgresql.adoc:108]:
```suggestion
```

**Review by maximiliankolb — COMMENTED** [review]
I don't fully understand which part is user input and which is (interactive) stdout.

**Comment by evgeni** [comment]
> I don't fully understand which part is user input and which is (interactive) stdout.

 I was afraid of that.

`<dbname>=#` is the normal PostgreSQL shell prompt.

So you do `\c pulpcore<enter>` which switches to the Pulp database, and then you do `CREATE EXTENSION ….;<enter>` which does the creation.

The non-prefixed lines are the ones returned by the DB.

**Comment by evgeni** [comment]
The alternative is to just print the commands, like
```
\c pulpcore
CREATE EXTENSION IF NOT EXISTS "hstore";
```

But it feels odd to me (with my PostgreSQL knowledge)

**Comment by ehelms** [comment]
We need to include the same step for upgrades. Are you thinking to handle that in a separate PR?

**Comment by mjivraja** [comment]
> @mjivraja @Lennonka any idea what this linkchecker error is all about? Safe to ignore?

This popped up right today. Looking at the link, this is a blocker.

**Comment by adamlazik1** [comment]
You can safely ignore that. I merged https://github.com/theforeman/foreman-documentation/pull/2564 without realizing that linkchecker now fails every build on master until 6.14 docs are published.

**Comment by evgeni** [comment]
It's unrelated https://github.com/theforeman/foreman-documentation/pull/2564#discussion_r1380021942

(And doesn't block publish anymore)

File: guides/common/modules/proc_installing-postgresql.adoc [hunk 2]
```diff
@@ -96,6 +96,15 @@ CREATE DATABASE candlepin OWNER candlepin;
...
 ----
 +
+. Connect to the Pulp database and create the `hstore` extension:
++
+[options="nowrap" subs="verbatim,quotes"]
+----
+postgres=# \c pulpcore
+You are now connected to database "pulpcore" as user "postgres".
+pulpcore=# CREATE EXTENSION IF NOT EXISTS "hstore";
+CREATE EXTENSION
```
Comment (mjivraja) [inline guides/common/modules/proc_installing-postgresql.adoc:106]:
```suggestion
----
+
You are now connected to database "pulpcore" as user "postgres".

. Create an `hstore` extension:
[options="nowrap" subs="verbatim,quotes"]
----
CREATE EXTENSION
```
Can this be better @maximiliankolb @evgeni ?

Comment (maximiliankolb) [reply guides/common/modules/proc_installing-postgresql.adoc:106]: Splitting commands sounds like a good idea to me.

Comment (evgeni) [reply guides/common/modules/proc_installing-postgresql.adoc]: updated, have a look please

**Review by mjivraja — APPROVED** [review]
LGTM. @maximiliankolb your final view?

**Review by maximiliankolb — APPROVED** [review]
Yes, makes sense to me.

**Comment by evgeni** [comment]
To github.com:theforeman/foreman-documentation.git
   06e11ebd7..ff922a334  3.6 -> 3.6
   e706193e2..9b8ec8d33  3.7 -> 3.7
   b4aca368a..705950d63  3.8 -> 3.8

**Pull request merged**